Program.cs

```diff
@@ -1,7 +1,51 @@
 ﻿namespace Training {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello World!");
+         Console.WriteLine ("Enter a password:");
+         string password = Console.ReadLine ();
+
+
+         bool Length = password.Length >= 6;

```

Comment:
```suggestion
         bool length = password.Length >= 6;
```
local variable names should start in lowercase